### PR TITLE
Improve javadocs for querydsl-codegen

### PR DESCRIPTION
**Pull request by timowest**
related to https://github.com/querydsl/querydsl/issues/1190

### CODE REVIEW

File: querydsl-codegen/src/main/java/com/querydsl/codegen/CodegenModule.java [hunk 1]
```diff
@@ -16,21 +16,36 @@
...
- * CodegenModule provides a module general serialization
+ * {@code CodegenModule} provides a module for general serialization
  *
  * @author tiwe
  *
  */
 public class CodegenModule  extends AbstractModule {
 
+    /**
+     * key for query type name prefix d
```
Comment (johnktims) [inline querydsl-codegen/src/main/java/com/querydsl/codegen/CodegenModule.java]: d?

File: querydsl-codegen/src/main/java/com/querydsl/codegen/ClassPathUtils.java [hunk 2]
```diff
@@ -13,28 +13,42 @@
...
  *
  * @author tiwe
  */
 public final class ClassPathUtils {
 
+    /**
+     * Return the classes from the given package and subpackages using the supplied classloader
+     *
+     * @param classLoader classloader to be used
+     * @param pkg package to be used
```
Comment (Shredder121) [inline querydsl-codegen/src/main/java/com/querydsl/codegen/ClassPathUtils.java]: `package to scan`?

File: querydsl-codegen/src/main/java/com/querydsl/codegen/ClassPathUtils.java [hunk 3]
```diff
@@ -13,28 +13,42 @@
...
+     */
     public static Set<Class<?>> scanPackage(ClassLoader classLoader, Package pkg) throws IOException {
         return scanPackage(classLoader, pkg.getName());
     }
 
+    /**
+     * Return the classes from the given package and subpackages using the supplied classloader
+     *
+     * @param classLoader classloader to be used
+     * @param pkg package to be used
```
Comment (Shredder121) [inline querydsl-codegen/src/main/java/com/querydsl/codegen/ClassPathUtils.java]: `package to scan`?

**Comment by timowest** [comment]
I added some improvements

**Pull request merged**